.semaphore/deploy-firefox.yml

```diff
@@ -0,0 +1,35 @@
+version: v1.0
+name: deploy-firefox
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu2004
+blocks:
+  - name: Deploy Firefox Extension
+    execution_time_limit:
+      minutes: 60
+    task:
+      secrets:
+        - name: github-pull-request-counter
+      prologue:
+        commands:
+          - checkout
+          - cache restore
+          - nvm install
+      jobs:
+        - name: Upload to Firefox Add-ons
+          commands:
+            - npm install --global web-ext
+            - |
+              cp -r src firefox-src
+              cat firefox-src/manifest.json | jq '.background = {"scripts": ["js/background.js"], "type": "module"}' > firefox-src/manifest.json.tmp
```

Comment:
Wouldn't it be easier to just store a `.patch` file and apply it?

Reply:
right, I agree. That would also make the diff of the chrome/firefox version more obvious. I'll add that